src/Adapter/Product/Combination/QueryHandler/GetEditableCombinationsListHandler.php

```diff
@@ -112,7 +114,7 @@ public function handle(GetEditableCombinationsList $query): CombinationListForEd
             $product,
             $combinations,
             $attributesInformation,
-            $this->combinationRepository->getTotalCombinationsCount($productId)
+            $this->combinationRepository->getTotalCombinationsCount($productId, $filters)
```

Comment:
This makes me think about something, for now each `EditableCombinationForListing` contains the associated attributes But when the filters will be implemented in the page we'll have to display all the attributes on top of the combination list So that they can be selected an used as filter values

So it might be interesting to have the whole list of attributes accessible in the `CombinationListForEditing` object, it would make the building of Attribute filters easier than if we have to loop through all the combination themselves

Besides since the `EditableCombinationForListing` only contains its own attributes, it means attributes associated to combination that are not in the current page won't even be accessible So in  a nutshell `CombinationListForEditing` should contain the list of all the attributes associated to all the combinations of the product (only those associated, we don't want the whole list of attributes here, only the revelant ones that we can filter on and be sure to get at least one result)

The list of attributes could very well be fetched from another query, but since most of them are already fetched in this one maybe it would be more convenient and more performant Killing two birds with one stone